Integrations/CentralBankIntegrationLib/CentralBankNewsPublisher.cs

```diff
@@ -19,19 +20,27 @@ public string GetNews()
             List<Valute> valutes = GetTopValutes();
 
             var sb = new StringBuilder();
-            if (valutes.Count == 0)
+            try

```

Comment:
do not use try catch in such case, it is not a solution for such small problem.
you can just check valutes != null

Reply:
fixed